Cargo.toml

```diff
@@ -21,30 +21,30 @@ edition = "2018"
 travis-ci = { repository = "poanetwork/hbbft" }
 
 [dependencies]
-bincode = "1.2.0"
-byteorder = "1.3.2"
-derivative = "2.0.2"
-env_logger = "0.7.1"
+bincode = "1.3.3"
```

Comment:
Since this is a minor upgrade you could theoretically just call `cargo update` on `minimint` without changing it here?

Reply:
I just ran `cargo upgrade`